src/cascadia/TerminalSettingsModel/ActionMap.cpp

```diff
@@ -649,7 +649,7 @@ namespace winrt::Microsoft::Terminal::Settings::Model::implementation
         }
 
         // Check our internal state.
-        const ActionAndArgs& actionAndArgs{ myAction, myArgs };
+        const auto actionAndArgs = winrt::make<ActionAndArgs>(myAction, myArgs);

```

Comment:
This can be written as the following (makes it more consistent with your other changes IMHO):
```cpp
const Model::ActionAndArgs actionAndArgs{ myAction, myArgs };
```

Reply:
I personally prefer using winrt::make here, especially since for it to be consistent with other changes, it should use `winrt::...`.

Reply:
Although I have to mention that you technically don't create an instance of `winrt::Microsoft::Terminal::Settings::Model::ActionAndArgs` here.
Instead you create an instance of `..::implementation::ActionAndArgs`, which is technically quite different from the rest of your changes (those create instances of the public interfaces). 😄

Reply:
Technically yes, however my aim was to change as little as possible, especially with regard to the types used. Changing from the implementation to the public interface does not seem necessary here to me :)